api/src/main/java/org/openmrs/OrderSet.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Represents the grouping of orders into a set,
+ * so as to give decision support for the doctors
+ */
+public class OrderSet extends BaseOpenmrsMetadata implements Serializable {
+	
+	public static final long serialVersionUID = 72232L;
+	
+	/**
+	 * Restrictions put on saving an orderSet.
+	 * ALL: All the members of the orderSet need to be selected for saving
+	 * ONE: Only one of the member of the orderSet needs to be selected for saving
+	 * ANY: Any of the members of the orderSet can be selected for saving
+	 */
+	public enum Operator {
+		ALL, ONE, ANY
+	};
+	
+	private Integer orderSetId;
+	
+	private Operator operator;
+
+	private Set<OrderSetMember> orderSetMembers;
+
+	/**
+	 * Gets the orderSetId
+	 *
+	 * @return the orderSetId
+	 */
+	public Integer getOrderSetId() {
+		return orderSetId;
+	}
+
+	/**
+	 * Sets the orderSetId
+	 *
+	 * @param orderSetId the orderSetId to set
+	 */
+	public void setOrderSetId(Integer orderSetId) {
+		this.orderSetId = orderSetId;
+	}
+
+	/**
+	 * Gets the operator
+	 *
+	 * @return the operator
+	 */
+	public Operator getOperator() {
+		return operator;
+	}
+
+	/**
+	 * Sets the operator
+	 *
+	 * @param operator the operator to set
+	 */
+	public void setOperator(Operator operator) {
+		this.operator = operator;
+	}
+
+	/**
+	 * Gets the orderSetMembers
+	 *
+	 * @return the orderSetMembers
+	 */
+	public Set<OrderSetMember> getOrderSetMembers() {
+		if (orderSetMembers == null) {
+			orderSetMembers = new HashSet<OrderSetMember>();
+		}
+		return orderSetMembers;
+	}
+
+	/**
+	 * Sets the orderSetMembers
+	 *
+	 * @param orderSetMembers the orderSetMembers to set
+	 */
+	public void setOrderSetMembers(Set<OrderSetMember> orderSetMembers) {
+		this.orderSetMembers = orderSetMembers;
+	}
+
+	public Integer getId() {
+		return getOrderSetId();
+	}
+	
+	public void setId(Integer id) {
+		setOrderSetId(id);
+	}
+	
+	public boolean equals(Object obj) {
```

Comment:
Removed the equals method from OrderSet.java, OrderGroup.java, OrderSetMember.java